src/pages/overview-page/catalog-item/index.tsx

```diff
@@ -0,0 +1,26 @@
+import React, { FC } from 'react';
+
+import { localization } from '../../../lib/localization';
+
+import { ConceptIcon, Title } from './styled';
+
+import './catalog-item.component.scss';
+
+interface ExternalProps {
+  itemsCount?: number;
+  linkUri: string;
+}
+
+interface Props extends ExternalProps {}
+
+export const CatalogItem: FC<Props> = ({ itemsCount, linkUri }) => (
+  <a className='catalog-item mb-2' href={linkUri}>
```

Comment:
Erstatte <a> med <Link> fra  import Link from '@fellesdatakatalog/link';
Erstatte "href" med "to" slik at man rutes til katalogen vha "react-router-dom".

Reply:
Helst også bruke styled-component og ikke bruke .scss -> fjerne alt av className.

Du kan definerer Link i styled.ts og ha stylingen der:

import Link from '@fellesdatakatalog/link';

const CatalogItem = styled(Link)`
...
`;

Her i komponenten blir det da:
<SC.CatalogItem to={catalogId}>

Reply:
48f3e62